finagle/buoyant/src/main/scala/com/twitter/finagle/buoyant/TlsClientConfig.scala

```diff
@@ -54,3 +54,14 @@ case class TlsClientConfig(
 }
 
 case class ClientAuth(certPath: String, keyPath: String)
+
+case class ClientTlsConfig(commonName: String, caCert: Option[String]) {
```

Comment:
I believe this config format is only used by the namerd thrift interpreter.  Why move it here?